src/components/spreadsheet/spreadsheet.ts

```diff
@@ -190,7 +190,14 @@ css/* scss */ `
     }
 
     .o-input[type="text"] {
-      border-width: 0 0 1px 0;
+      &:not([disabled]) {
+        border-width: 0 0 1px 0;
+      }
+
+      &:disabled {
+        border-width: 0 0 0 0;
+        cursor: not-allowed;
```

Comment:
Either put all the `disabled` css in the selection input css, or all the css here, I don't think there's a point of having half the css in each place